### DISCUSSION TITLE
Loki/Prometheus query builder: Make the raw query editable

### DISCUSSION THREAD

**Discussion by ivanahuckova** [grafana/grafana#52098] [Observability]
We could probably do this in 2 steps: 
1. Add an icon that leads you to the code editor. 
3. Re-think if separate code/builder/explain make sense, or if it should be 1 editor with toggleable modes.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
